src/views/v2/components/loader/loader.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * View: Loader
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/components/loader/loader.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @version TBD
+ */
+
+if ( empty( $text ) ) {
+	$text = $this->get( 'text' ) ?: esc_html__( 'Loading...', 'event-tickets' );
```

Comment:
are you using `$text` in this template?

Reply:
Not now for RSVP. But the purpose of _componetizing_ this in `v2/` is to organise everything that is v2 there, so I took the other version as a reference. Where text is a thing.

Reply:
Well, I just re-checked the template and you can forget of what I just said haha. We're not using it on the other template. I'll remove it.